docs/release_process_quarterly.md

```diff
@@ -0,0 +1,501 @@
+        - [ ] test stop line stop events
+
+
+## jpo-conflictvisualizer
+### Prerequisites
+    - [ ] jpo-ode release completed
+    - [ ] jpo-geojsonconverter release completed
+    - [ ] jpo-conflictmonitor release completed
+
+### 1. Code Ready & Release Notes
+    - [ ] Release notes drafted & added to `Release_notes.md` file in `docs` directory
+    - [ ] Code changes for release are merged into `develop`
+    - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+### 2. Preliminary Testing
+    - [ ] code compiles
+    - [ ] unit tests pass
+    - [ ] program starts up correctly
+    - [ ] GUI functions & can display messages
+
+### 3. Project Reference Updates & Release Creation
+    - [ ] Update version number in pom.xml file for the `jpo-conflictvisualizer` project if not already done. The pom.xml can be found in the `api/jpo-conflictvisualizer-api` directory.
+    - [ ] Update git submodule & artifact references for the 'jpo-conflictvisualizer' project.
+        - [ ] Open the jpo-conflictvisualizer project in an IDE.
+        - [ ] Navigate to the jpo-utils directory and run `git checkout tags/jpo-utils-x.x.x` to update the submodule reference.
+        - [ ] Update the version number in the api/jpo-conflictvisualizer-api/pom.xml file for the jpo-geojsonconverter, jpo-ode-*, and jpo-conflictmonitor dependencies to match the version number of the corresponding releases. (e.g. 1.0.0)
+        - [ ] Commit these changes to the `release/(year)-(quarter)` branch & push the changes to the remote repository.
+        - [ ] Ensure these changes pass CI/CD checks before continuing.
+    - [ ] Merge `release/(year)-(quarter)` branch into `master` branch for the jpo-conflictvisualizer project, and create a release with the version number of the release. (e.g. jpo-conflictvisualizer-x.x.x)
+    - [ ] Create docker images
+        - [ ] Use the release for the jpo-conflictmonitor to produce docker images containing the version number.
+            - [ ] One image will be for the API
+            - [ ] One image will be for Keycloak
+        - [ ] Upload docker images to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker images with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker images with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker images with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] jpo-conflictvisualizer-api
+        - [ ] image starts up correctly
+        - [ ] GUI functions & can display messages
+    - [ ] jpo-conflictvisualizer-keycloak
+        - [ ] image starts up correctly
+        - [ ] authentication verified to work
+
+
+## jpo-deduplicator
+### Prerequisites
+    - [ ] jpo-ode release completed
+    - [ ] jpo-geojsonconverter release completed
+    - [ ] jpo-conflictmonitor release completed
```

Comment:
Nature of dependencies:

    - [ ] jpo-ode (Maven, Dockerhub) release completed
    - [ ] jpo-geojsonconverter (Maven, Dockerhub) release completed
    - [ ] jpo-conflictmonitor (Maven, Dockerhub) release completed

Reply:
Addressed!